src/Icons/doc/index.rst

```diff
@@ -19,10 +19,60 @@ Installation
 
     $ composer require symfony/ux-icons
 
-Icons?
+Icons
 ------
 
-No icons are provided by this package but there are several ways to include and render icons.
+The ``symfony/ux-icons`` package aspires to offer simple and easy-to-use ways to
+handle SVG icons in your Symfony application.
+
+SVG Icons
+~~~~~~~~~
+
+[SVG](https://en.wikipedia.org/wiki/SVG) (Scalable Vector Graphics) is an XML-based
```

Comment:
Docs use RST instead of MD, so links are different:

```md
[SVG](https://en.wikipedia.org/wiki/SVG) ...
```

```rst
`SVG`_ ...

// then, at the bottom of the file, add this:
.. _`SVG`: https://en.wikipedia.org/wiki/SVG
```

Reply:
100% forgot i was not writing mardown 😅